### ISSUE TITLE
Merge Opportunity?

### ISSUE THREAD

**Issue by nbouvrette** [calibr/properties-file#1]
Hi @calibr 

I just released a brand new package written in TypeScript that mimics exactly the Java behavior for .properties. files: https://www.npmjs.com/package/properties-files

I see you haven't updated your npm package for a while and a lof the good package name for properties files are taken already so I was wondering if maybe you might be interested in possibly merging our 2 packages?

Thanks,
Nicolas

**Comment by calibr** [comment]
@nbouvrette Ok, let's do that if API will be compatible with with the current properties-file package.

**Comment by nbouvrette** [comment]
I was actually thinking of releasing a v2.0 that would include breaking changes (not compatible) and keep the current API: https://github.com/Avansai/properties-files/blob/main/README.md

Let me know if you think it would be an option

**Comment by calibr** [comment]
@nbouvrette Ok, let's do that.

**Comment by nbouvrette** [comment]
Awesome! can you provide me maintainer access to your npm package? I can probably handle all the versioning on my end once I have access - I'll add a mention of what is going on, on the readme file. This is my npm profile: https://www.npmjs.com/~nbouvrette

**Comment by nbouvrette** [comment]
Thanks! I'll try to make the switch as soon as possible (might have to happen over next weekend)

**Comment by nbouvrette** [comment]
I just completed the release of version 2.0.1 which points to our repo. I hope you find the special mention appropriate in the readme? I also added a note on top as a warning if anyone was using the previous version which should avoid confusion.

I removed your contributor access for now since you don't have access to the new repository - let me know if you would like to contribute back to this project and I can grant you back access - feel free to ping me on Twitter [@nicolasbou](https://twitter.com/nicolasbou)

Thanks again for helping make our package easier to find!
